packages/reassure-compare/src/compare.ts

```diff
@@ -100,13 +108,40 @@ async function loadFile(path: string): Promise<PerformanceRecord> {
   return result;
 }
 
+/**
+ * Load performance file and parse it to `MetadataEntry` object.
+ */
+async function loadMetadataFromFile(path: string): Promise<MeasurementMetadata> {
+  const entries: MetadataEntry[] = await readFileAndParseDataToFormat(path, 'metadata');
+
+  if (entries.length > 1) {
+    const msg = `Your performance result file ${path} contains records with more than one metadata lines.
+      Please ensure to have only one metadata entry in the perf report file
+      `;
+    logError(msg);
+    throw new Error(msg);
+  }
+  if (entries.length == 0) {
```

Comment:
I think we should allow for missing metadata, as this will be the case when running baseline calculation after interoducing metadata features, the baseline will still use the previous Reassure and will not insert that entry.